django/apps/config.py

```diff
@@ -91,7 +91,7 @@ def create(cls, entry):
         app_config_class = None
         app_name = None
         app_module = None
-
+        app_config_name = None
```

Comment:
Might want to move this line just after `app_config_class = None` and keep the blank line, which helps a bit the readability of this way-too-long function.

Reply:
Done @aaugustin!